NOTICE.md

```diff
@@ -23,9 +23,25 @@ information.
 
 There are third-party content used by and shipped with this project with different licenses.
 
-This project uses Gradle and the Gradle Wrapper
+### Gradle & Gradle Wrapper
+This project uses Gradle and the Gradle Wrapper.
 - GitHub project: https://github.com/gradle/gradle
-- License: [Apache-2.0 license](https://github.com/gradle/gradle/blob/master/LICENSE)
+- License: [Apache-2.0 license](https://github.com/gradle/gradle/blob/8a9cda36b91f1b7f66d0c2c27e5594a210bac8f3/LICENSE)
+
+### unDraw Illustrations
+This project uses illustrations from unDraw.
+- Source URL: https://undraw.co/illustrations
+- License: https://undraw.co/license
+
+### Docusaurus
+The documentation uses the Docusaurus framework with its autogenerated files.
+- GitHub project: https://github.com/facebook/docusaurus
+- License (code): [MIT license](https://github.com/facebook/docusaurus/blob/6c7f2fde20b2448a1b966dd5f674205c0a08eeba/LICENSE)
```

Comment:
Is it legally correct to link against a fixed version and not the current up-to-date file from the main branch? (e.g., https://github.com/facebook/docusaurus/blob/main/LICENSE)

Reply:
IMHO, there is no correct way.

On the one hand, using such a fixed permalink ensures that the used version of the license is correct. Further, if the project updates the license for a newer version but we stay on the older version, the link to the license would be wrong. Additionally, if the name of the main branch or the location of the `LICENSE` file changes, a non-permalink would result in a 404 error.

On the other hand, the NOTICE file should point to the current LICENSE in the latest version. To reduce the maintenance, I followed your advice to change the link to the latest version.